controller/src/server.rs

```diff
@@ -0,0 +1,71 @@
+use tonic::{transport::Server, Request, Response, Status};
+
+use scheduler::scheduling_service_server::{SchedulingService, SchedulingServiceServer};
+use scheduler::{SchedulingRequest, SchedulingResponse};
+use axum::{response::Html, routing::get, Router};
+use std::net::SocketAddr;
+use tokio::task;
+
+pub mod scheduler {
+    tonic::include_proto!("orkascheduler"); // The string specified here must match the proto package name
+}
+
+#[derive(Debug, Default)]
+pub struct MySchedulingService {}
+
+#[tonic::async_trait]
+impl SchedulingService for MySchedulingService {
+    async fn schedule(
+        &self,
+        request: Request<SchedulingRequest>,
+    ) -> Result<Response<SchedulingResponse>, Status> {
+        println!("Got a request: {:?}", request);
+
+        let response = SchedulingResponse {
+            status_code: 0,
+            rejection_reason: Some(1),
+        };
+
+        Ok(Response::new(response))
+    }
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // GRPC
```

Comment:
That's not a really useful comment, imo. Something like `Initialize the gRPC server` would be better.

Reply:
Done :+1: